docs/_includes/Types.md

```diff
@@ -480,25 +493,61 @@ code.  Co-inductive datatypes, function types, as well as inductive
 datatypes with ghost parameters are examples of types that are not
 equality supporting.
 
-TO BE WRITTEN: There are now more such modes
+## Auto-initializable types: `T(0)`
 
-[^fn-type-mode]:  Being equality-supporting is just one of many
-    _modes_ that one can imagine types in a rich type system to have.
-    For example, other modes could include having a total order,
-    being zero-initializable, and possibly being uninhabited.  If
-    Dafny were to support more modes in the future, the `( )`-suffix
-    syntax may be extended.  For now, the suffix can only indicate the
-    equality-supporting mode.
+All Dafny variables of a given type hold a legal value of that type;
+if no explicit initialization is given, then an arbitrary value is
+assumed.
```

Comment:
This could possible say "assumed by the verifier and provided by the compiler".

Reply:
OK